hack/update-goreleaser-yaml.sh

```diff
@@ -16,9 +21,14 @@ if [[ $release_preflight == true ]]; then
 fi
 
 if [[ $release_log_collector != true ]]; then
-  echo "removing log-collector packages from goreleaser.yml"
+  echo "skip log-collector packages release from goreleaser.yml"
   sed -i '/binary: log-collector/a \ \ skip: true' "$goreleaser_yaml"
 fi
 
+#if [[ $release_target_browser != true ]]; then
```

Comment:
We don't need a way to skip the target browser plugin release?

Reply:
We'll have to manage different versions and different release cycle for preflight, log-collector & target-browser so need these changes.